playbooks/common/openshift-cluster/upgrades/docker/docker_upgrade.yml

```diff
@@ -52,13 +52,19 @@
 
   - name: Drain Node for Kubelet upgrade
     command: >
-      {{ openshift.common.admin_binary }} drain {{ openshift.node.nodename }} --config={{ openshift.common.config_base }}/master/admin.kubeconfig --force --delete-local-data --ignore-daemonsets
+      {{ hostvars[groups.oo_first_master.0].openshift.common.admin_binary }} drain {{ openshift.node.nodename | lower }}
```

Comment:
I don't think you need `hostvars[groups.oo_first_master.0]` as its being delegated already

Reply:
will change it

Reply:
I think we want hostvars here; if the master is RHEL and the node is atomic, this would fail otherwise.

Reply:
now i learnt s'thing new, cheers @michaelgugino ;)